src/mkinitcpio.rs

```diff
@@ -0,0 +1,139 @@
+use crate::parse_args::AppConfig;
+use crate::utils::csleep;
+use rand::prelude::*;
+use rand::seq::SliceRandom;
+use yansi::Paint;
+
+const ALL_HOOKS: &'static [&str] = &[
```

Comment:
Do you think there's merit in putting these into a list like [here](https://github.com/svenstaro/genact/blob/master/src/main.rs#L52)? Might make it more consistent and we could also reuse these hooks for some other modules or mix them in as technical mumbo-jumbo when generating some other stuff.

Reply:
All done.